service/chain/disk.go

```diff
@@ -164,6 +164,27 @@ func (d *Disk) Transactions(height uint64) ([]*flow.TransactionBody, error) {
 	return transactions, nil
 }
 
+func (d *Disk) Results(height uint64) ([]*flow.TransactionResult, error) {
+	blockID, err := d.block(height)
+	if err != nil {
+		return nil, fmt.Errorf("could not get block for height: %w", err)
+	}
+
+	var txRes []flow.TransactionResult
```

Comment:
Still would prefer to use `results` here, because `txRes` hides the fact it's plural.